frontend/src/lib/Home.svelte

```diff
@@ -0,0 +1,6 @@
+<script>
+  import { replace } from 'svelte-spa-router'
+  import languages from '../languages/languages.json'
+
+  replace('/et/')
```

Comment:
Why did you hardcode `et`?

Reply:
![image](https://user-images.githubusercontent.com/1296558/180002305-1b69dba8-f4f0-4b84-81d1-5f4336ee4f7a.png)